homeassistant/components/meater/sensor.py

```diff
@@ -146,13 +146,13 @@ def async_update_data():
         if not coordinator.last_update_success:
             return
 
-        devices = coordinator.data
+        devices: dict[str, MeaterProbe] = coordinator.data
         entities = []
         known_probes: set = hass.data[DOMAIN]["known_probes"]
 
         # Add entities for temperature probes which we've not yet seen
         for dev in devices:
```

Comment:
It would be more clear if we rename the variable to `device_id` instead of `dev`. We call it `device_id` in the entity.

Reply:
That's much clearer, implemented in https://github.com/home-assistant/core/pull/71333 👍